### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by blog2020** [XTLS/Xray-core#3928] [General]
搞错了

**Comment by RPRX** [comment]
多少有点搞笑，你的 SS 端口被墙，和你入站是 REALITY、或者入站是什么都没有任何关系，你描述的情况可能性只有：

1. 你直连了 SS，经过了 GFW
2. 你的 IP 有黑历史，GFW 持续扫描

我猜是第一种情况，毕竟你不直连的话怎么知道 SS 端口被 GFW 墙了？况且仅境外根本不会关心 GFW 墙没墙，今日笑话预定。
